test/test_librosa_compatibility.py

```diff
@@ -91,18 +92,18 @@ def test_amplitude_to_DB(self):
 
         ta_out = F.amplitude_to_DB(spec, multiplier, amin, db_multiplier, top_db)
         lr_out = librosa.core.power_to_db(spec.numpy())
-        lr_out = torch.from_numpy(lr_out).unsqueeze(0)
```

Comment:
does this have to do with the assert change?

Reply:
Yes, the shapes are not matching in the original test. Now with `assert_allclose` that is causing the issue so it has to be removed.

Reply:
sounds good!

Reply:
sounds good!